specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2021-06-01/application.json

```diff
@@ -0,0 +1,2285 @@
+        }
+      }
+    },
+    "UserAssignedIdentityMap": {
+      "type": "object",
+      "description": "The list of user identities associated with the resource. The user identity dictionary key references will be ARM resource ids in the form:\n'/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.\n",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserAssignedIdentity"
+      }
+    },
+    "UniformInt64RangePartitionSchemeDescription": {
+      "description": "Describes a partitioning scheme where an integer range is allocated evenly across a number of partitions.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/PartitionSchemeDescription"
+        },
+        {
+          "type": "object",
+          "description": "UniformInt64RangePartitionSchemeDescription"
+        }
+      ],
+      "x-ms-discriminator-value": "UniformInt64Range",
+      "required": [
+        "count",
+        "lowKey",
+        "highKey"
+      ],
+      "properties": {
+        "count": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of partitions."
+        },
+        "lowKey": {
+          "type": "string",
+          "description": "String indicating the lower bound of the partition key range that\nshould be split between the partition ‘count’\n"
+        },
+        "highKey": {
+          "type": "string",
+          "description": "String indicating the upper bound of the partition key range that\nshould be split between the partition ‘count’\n"
+        }
+      }
+    },
+    "UpgradeDomainTimeout": {
+      "type": "string",
+      "description": "The amount of time each upgrade domain has to complete before FailureAction is executed. It is first interpreted as a string representing an ISO 8601 duration. If that fails, then it is interpreted as a number representing the total number of milliseconds.",
+      "default": "P10675199DT02H48M05.4775807S"
+    },
+    "UpgradeTimeout": {
+      "type": "string",
+      "description": "The amount of time the overall upgrade has to complete before FailureAction is executed. It is first interpreted as a string representing an ISO 8601 duration. If that fails, then it is interpreted as a number representing the total number of milliseconds.",
+      "default": "P10675199DT02H48M05.4775807S"
+    },
+    "ErrorModel": {
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModelError"
+        }
+      },
+      "description": "The structure of the error."
+    },
+    "ErrorModelError": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "The error details."
+    },
+    "SystemData": {
```

Comment:
>SystemData [](http://example.com/codeflow?start=5&length=10)

Please refer common definition from [https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json)

Reply:
@chiragg4u when I refer to the common type I start getting these errors: https://github.com/Azure/azure-rest-api-specs/pull/14838/checks?check_run_id=2850749457. Am I doing something wrong?